### PR TITLE
Add vlanId to bridge interior port

### PR DESCRIPTION
**Pull request by srvaroa**
Both to Python client and CLI

Signed-off-by: Galo Navarro galo@midokura.com

### CODE REVIEW

File: src/bin/mn-ctl [hunk 1]
```diff
@@ -1536,6 +1536,11 @@ class BridgePort(ObjectType):
         self.put_attr(SingleAttr(name = 'outfilter',
                                  value_type = ObjectRef('chain'),
                                  getter = 'get_outbound_filter_id'))
+        self.put_attr(SingleAttr(name = 'vlanid',
```
Comment (gontanon) [inline src/bin/mn-ctl]: Can we name this just 'vlan' to be more consistent with CLI naming?

Comment (srvaroa) [inline src/bin/mn-ctl]: Done

Comment (gontanon) [inline src/bin/mn-ctl]: Thx!

**Comment by tomoe** [comment]
@srvaroa is this request still active? do we need more changes for l2gw e.g. exterior port type?

**Comment by srvaroa** [comment]
Hi Tomoe,

This is still valid. #11 and #12 were old and included more changes for
exterior ports and the new VlanAwareBridge device, but in the second
pass we deprecated that one and chose not to publish an API for those.

These should be the only changes still needed.

Thanks!

**Comment by tomoe** [comment]
@srvaroa thanks for the explanation. Actually @takayukiu was able to use this to test l2gw on his mmm. So merging...

**Pull request merged**